QueAns/migrations/0005_auto_20180206_0740.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.2 on 2018-02-06 07:40
```

Comment:
In one PR, always have a single migration.

Why? 
- Migrations are a way of telling DB that these are the changes we want to have in DB state, so generally, every PR should have only one migration to tell DB to know of changes. If we have multiple migrations we would have un-necessary migration tree on the production server which is really not needed.
- Suppose that you have multiple migration 0002, 0003 and 0004 and now in 0002 you were adding a column, in 0003 you were deleting one and 0004 you were again adding. So, final migration effect would be only adding the column, but due to we have multiple migrations on the production server would do the same in order which is un-necessary and maybe it will induce errors.

Reply:
Okay